.github/workflows/build.yml

```diff
@@ -2,52 +2,18 @@ name: CI
 
 on:
   push:
-    branches: [ main ]
-  workflow_dispatch:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  release:
+    types: [published]
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
-    steps:
-      -
-        name: Checkout
-        uses: actions/checkout@v2
-      -
-        name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
-      -
-        name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
-      -
-        name: Login to DockerHub
-        if: github.ref == 'refs/heads/main'
-        uses: docker/login-action@v2
-        with:
-          username: mekomsolutions
-          password: ${{ secrets.DOCKER_HUB_REGISTRY_PASSWORD }}
-      -
-        name: Docker meta
-        id: meta
-        uses: docker/metadata-action@v4
-        
-        with:
-          images: |
-            mekomsolutions/senaite
-          tags: |
-            type=raw,value=latest,enable={{is_default_branch}}
-            type=pep440,pattern={{version}}
-            type=pep440,pattern={{major}}.{{minor}}
-            type=sha
-      - 
-        name: Get commit id
-        id: vars
-        run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
-      -
-        name: Build and push
-        uses: docker/build-push-action@v2
-        env: 
-          SERVICE: senaite
-        with:
-          platforms: linux/amd64,linux/arm64
-          push: true
-          tags: ${{ steps.meta.outputs.tags }}
+  docker-release:
+    uses: mekomsolutions/shared-github-workflow/.github/workflows/docker-build-publish.yml@main
+    with:
+      download-artifacts: false
+      image-name: "senaite"
+    secrets:
+      DOCKER_HUB_USERNAME: mekomsolutions
```

Comment:
Let's use username from secrets here.

Reply:
I am not sure we have one. We will need @rbuisson  to confirm